tools/abel/setup.py

```diff
@@ -26,5 +26,5 @@
     install_requires=[
         'absl-py', 'pyfiglet', 'argparse', 'pyyaml', 'google-api-python-client'
     ],
-    python_requires='>=3.9',
+    python_requires='>=3.11',
```

Comment:
those are ok to keep at >3.9
since the 3.9 is supported until 2025
Please update only the github actions to make the tests go faster, since 3.11 introduces performance improvements 
https://www.python.org/downloads/

Reply:
3.9 security 2020-10-05 2025-10 [PEP 596](https://www.python.org/dev/peps/pep-0596)